File: Java/Mathematics/trailing-zeros-in-factorial.java
import java.io.*;
import java.util.*;
import static java.lang.System.out;

class TrailingZerosInFactorial {
    static int countTrailingZeros(int number) {
        int res = 0;
        for(int i = 5; i <= number; i = i * 5) {
            res = res + (number / i);
        }
        return res;
    }

    public static void main(String[] args) {
        System.out.println(countTrailingZeros(251));
    }
}

File: C++/Mathematics/trailing-zeros-in-factorial.cpp
#include <iostream>
#include <limits.h>

using namespace std;

int countTrailingZeros(int number) {
    int res = 0;
    for(int i = 5; i <= number; i = i * 5) {
        res = res + (number / i);
    }
    return res;
}

int main() {
    cout << countTrailingZeros(251);
}